cypress/elements/dashboardFilter.js

```diff
@@ -0,0 +1,6 @@
+export const filterBadgeSel = '[data-test="dashboard-filter-badge"]'
```

Comment:
`..Sel` not `..El` ? 🤔

Reply:
El = element,
Sel = selector?
But the file is in /elements/....

Reply:
Changed dir name to selectors